src/utils/types.ts

```diff
@@ -174,6 +174,7 @@ export interface BlogMetaType {
 }
 
 export interface IBlog {
+    description: string | undefined;
```

Comment:
Make optional

Reply:
Do you mean change the syntax?

```
- description: string | undefined
+ description?: string
```

Reply:
@gixxerblade, it's already optional. If you delete the JSON section that would house the description, it will not break.

Reply:
@mrbernnz I didn't, I will change that in another branch or if you could check out the branch and change it I will approve it.